README.md

```diff
@@ -1,2 +1,27 @@
 # Python User Language Support
-Python User Language Support for [Cloudstate](https://github.com/cloudstateio/cloudstate).
\ No newline at end of file
+Python User Language Support for [Cloudstate](https://github.com/cloudstateio/cloudstate).
+
+## Installation via source
+
+```
+> git clone https://github.com/cloudstateio/python-support.git
+Cloning into 'python-support'...
+
+> cd python-support
+> python3 -m venv ./venv 
+> source ./venv/bin/activate
+> python --version     
+Python 3.7.3
+> pip --version 
+> pip install -r requirements.txt
+```
```

Comment:
I cloned a repo and after this step, I could not start `./tck.sh` but after `pip install .` I where.

Reply:
@sleipnir wdyt?